package.json

```diff
@@ -49,35 +40,33 @@
   "author": "Ben Coe <ben@npmjs.com>",
   "license": "ISC",
   "dependencies": {
-    "string-width": "^4.2.0",
-    "strip-ansi": "^6.0.1",
-    "wrap-ansi": "^7.0.0"
+    "string-width": "^7.2.0",
+    "strip-ansi": "^7.1.0",
+    "wrap-ansi": "^9.0.0"
   },
   "devDependencies": {
-    "@types/node": "^14.0.27",
+    "@types/node": "^22.13.10",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
     "@typescript-eslint/parser": "^4.0.0",
-    "c8": "^7.3.0",
-    "chai": "^4.2.0",
-    "chalk": "^4.1.0",
+    "c8": "^10.1.3",
+    "chai": "^5.2.0",
+    "chalk": "^5.4.1",
     "cross-env": "^7.0.2",
     "eslint": "^7.6.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-node": "^11.1.0",
-    "gts": "^3.0.0",
-    "mocha": "^10.0.0",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.23.1",
-    "rollup-plugin-ts": "^3.0.2",
+    "gts": "^6.0.2",
+    "mocha": "^11.1.0",
+    "rimraf": "^6.0.1",
     "standardx": "^7.0.0",
-    "typescript": "^4.0.0"
+    "typescript": "^5.8.2"
   },
   "files": [
     "build",
     "index.mjs",
     "!*.d.ts"
   ],
   "engines": {
-    "node": ">=12"
+    "node": ">=20"
```

Comment:
I'm bumping into some issues in yargs with `Node@<20.19.0`, should we consider making this our base engine version?

Reply:
Is 20.19.0 around something in particular? require(esm) ?

In which case, we might want a minimum for node 22 as well.

I am not against the idea of minimum versions within a major when there is a reason. (I need to remind myself that the early minor versions were before LTS status, so clearly not all equal! 😆 )

Reply:
I didn't notice this had been merged when I made this command. No reply necessary!